packages/grafana-ui/src/components/Combobox/ComboboxList.tsx

```diff
@@ -0,0 +1,153 @@
+            const rowItem = options[row.index];
+            return rowItem.group === item.group;
+          });
+          const groupHeaderItem = groupHeaderIndex && options[groupHeaderIndex.index];
+
+          const itemId = `combobox-option-${item.value}`;
+          // If we're rendering the group header, this is the ID for it. Otherwise its used on
+          // the option for aria-describedby.
+          const groupHeaderId = groupHeaderItem ? `combobox-option-group-${groupHeaderItem.value}` : undefined;
+
+          return (
+            // Wrapping div should have no styling other than virtual list positioning.
+            // It's children (header and option) should appear as flat list items.
+            <div
+              key={item.value}
+              className={styles.listItem}
+              style={{
+                height: virtualRow.size,
+                transform: `translateY(${virtualRow.start}px)`,
+              }}
+            >
+              {/* Group header */}
+              {startingNewGroup && (
+                <div role="presentation" id={groupHeaderId} className={styles.optionGroupHeader}>
+                  {item.group}
+                </div>
+              )}
+
+              {/* Option */}
+              <div
+                className={cx(
+                  styles.option,
+                  !isMultiSelect && isOptionSelected(item) && styles.optionSelected,
+                  highlightedIndex === virtualRow.index && styles.optionFocused
+                )}
+                {...getItemProps({
+                  item: item,
+                  index: virtualRow.index,
+                  id: itemId,
+                  'aria-describedby': groupHeaderId,
+                })}
+              >
+                {isMultiSelect && (
+                  <div className={styles.optionAccessory}>
+                    <Checkbox
+                      key={itemId}
+                      value={allItemsSelected || isOptionSelected(item)}
+                      indeterminate={item.value === ALL_OPTION_VALUE && selectedItems.length > 0 && !allItemsSelected}
+                      aria-labelledby={itemId}
+                      onClick={(e) => {
+                        e.stopPropagation();
+                      }}
+                    />
+                  </div>
+                )}
+
+                {/* <div className={styles.optionAccessory}>
+                  <Icon name="external-link-alt" />
+                </div> */}
+
+                <div className={styles.optionBody}>
+                  {/* TODO: need to add back the 'select all' thing */}
+                  <div className={styles.optionLabel}>{item.label ?? item.value}</div>
+
+                  {item.description && <div className={styles.optionDescription}>{item.description}</div>}
+                </div>
+              </div>
+            </div>
+          );
+        })}
+      </div>
+
+      {/* TODO: missing other messages, like async error? */}
```

Comment:
I guess these 'select all' and 'async error' are going to be added in following PRs, right?

Reply:
No, they must be added into this PR so it doesn't regress.

Reply:
🤔 So, then, this is not in review? Or do you want us to review this stage?